sources/install/package_crypto.sh

```diff
@@ -40,6 +40,14 @@ function install_rsacracker() {
     add-to-list "RsaCracker,https://github.com/skyf0l/RsaCracker,Powerful RSA cracker for CTFs. Supports RSA - X509 - OPENSSH in PEM and DER formats."
 }
 
+function install_sage {
+    colorecho "Installing Sage"
```

Comment:
you'll need to add a CCC exclusion here as per https://exegol.readthedocs.io/en/latest/community/contributors.html#function-structure
<img width="729" alt="image" src="https://github.com/user-attachments/assets/35e015c9-e094-4720-8652-3fcc591587c7">
For `add-aliases`